### PR TITLE
refactor: remove replica id

### PR DESCRIPTION
**Pull request by JohannesMessner**

### CODE REVIEW

File: docs/fundamentals/executor/executor-built-in-features.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ different purposes.
 
 - **`.metas` are statically defined.** "Static" means, e.g. from hard-coded value in the code, from a YAML file.
 - **`.runtime_args` are dynamically determined during runtime.** Means that you don't know the value before running
-  the `Executor`, e.g. `pea_id`, `replicas`, `replica_id`. Those values are often related to the system/network
+  the `Executor`, e.g. `pea_id`, `replicas`. Those values are often related to the system/network
```
Comment (JoanFM) [inline docs/fundamentals/executor/executor-built-in-features.md:198]: can u please change pea_id to shard_id? I forgot to change in my old PR

Comment (JohannesMessner) [reply docs/fundamentals/executor/executor-built-in-features.md:198]: Sure thing

File: jina/peapods/pods/__init__.py [hunk 2]
```diff
@@ -750,7 +745,6 @@ def _set_peas_args(args: Namespace) -> Dict[int, List[Namespace]]:
             for replica_id in range(args.replicas):
                 _args = copy.deepcopy(args)
                 _args.shard_id = shard_id
-                _args.replica_id = replica_id
                 _args.pea_role = PeaRoleType.WORKER
                 _args.identity = random_identity()
```
Comment (JohannesMessner) [inline jina/peapods/pods/__init__.py:750]: It doesn't let me comment directly there, but just below here is some more code that seems to be related to replica_id, but I am not sure what it does. Does it need to be removed/changed?

Comment (JoanFM) [reply jina/peapods/pods/__init__.py:750]: does not seem so

**Review by JoanFM — APPROVED** [review]

**Pull request merged**